wtxmgr/tx.go

```diff
@@ -367,6 +367,7 @@ func (s *Store) PruneOldTickets(ns walletdb.ReadWriteBucket) error {
 type Store struct {
 	chainParams    *chaincfg.Params
 	acctLookupFunc func(walletdb.ReadBucket, dcrutil.Address) (uint32, error)
+	acctToNameFunc func(walletdb.ReadBucket, uint32) (string, error)
```

Comment:
Please don't make this worse.  Instead, the caller of these balance functions should use the returned account number to look up the account names.